hypha/apply/funds/tests/test_views.py

```diff
@@ -329,6 +329,7 @@ def test_can_create_project(self):
                 "project_create_form": "",
                 "project_lead": self.user.id,
                 "project_initial_status": CONTRACTING,
+                "project_end": timezone.now(),
```

Comment:
```suggestion
                "project_end": timezone.now().strftime("%Y-%m-%d"),
```
From what I saw locally only using `timezone.now()` was causing the form's `DateField` to be invalid